xee/ext.py

```diff
@@ -527,7 +528,9 @@ def _get_primary_coordinates(self) -> list[Any]:
       )
     if self.primary_dim_property in ['system:time_start', 'system:time_end']:
       # Convert elements in primary_dim_list to np.datetime64
```

Comment:
Could you update this comment?

Something like `# Convert elements in primary_coords to a timestamp.`.

Reply:
Comment updated.